### PR TITLE
Edit articles

### PR DESCRIPTION
**Pull request by koggs**
### What we did:
- Create edit view with a form
- Create link to edit article in show view
- Modify article controller

### CODE REVIEW

**Review by GeorgeJLewis — APPROVED** [review]
Looks good to me

File: app/views/articles/edit.html.haml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+= form_with model: @article , local: true do |form|
+  = form.label :headline
+  %br
+  = form.text_field :headline
+  %br
+  = form.label :content
+  %br
+  = form.text_field :content
+  %br
+  = form.submit 'Save Article'
```
Comment (magnus-thor) [inline app/views/articles/edit.html.haml]: You have this same form on new.html.haml, extract the forms to a partial and render the partial on each page

File: features/edit_articles.feature [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+Scenario: Journalist edits an article
+  Then I should see "Edit Article"
+  And I should see "The awesome article" in "article headline" field
+  And I should see "Lorem ipsum dolor sit amet" in "article content" field
+  When I fill in "Headline" with "The modified article"
+  And I fill in "Content" with "This is the modified text content"
+  And I click "Save Article"
+  Then I should see "Article successfully saved and sent for approval"
+
+  Scenario: Journalist leaves headline empty
```
Comment (magnus-thor) [inline features/edit_articles.feature]: Indentation is off

File: features/edit_articles.feature [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    When I fill in "Headline" with ""
+    And I click "Save Article"
+    Then I should see "Headline can't be blank"
+
+  Scenario: Journalist leaves content empty
```
Comment (magnus-thor) [inline features/edit_articles.feature]: Indentation is off

**Review by magnus-thor — CHANGES_REQUESTED** [review]

**Comment by alghazu** [comment]
@magnus-thor Addressed comments

**Comment by alghazu** [comment]
@magnus-thor solved merge conflicts

**Review by magnus-thor — APPROVED** [review]
Looks good

**Pull request merged**